code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -16,6 +16,8 @@
 	hud_type = /datum/hud/ai
 	buckle_flags = NONE
 	has_unlimited_silicon_privilege = TRUE
+	///are we already firing a railgun laser
+	var/is_laser_firing = FALSE
```

Comment:
Could make this a trait

Reply:
Done, checks are failing because CHAT_FILTER_CHECK is missing. I have no idea why it's missing, I haven't touched it, but it's failing on multiple PRs other than this one.